gradle/dependencies.gradle

```diff
@@ -54,7 +54,10 @@ versions += [
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
   bcpkix: "1.78.1",
-  caffeine: "2.9.3", // 3.x supports JDK 11 and above
+  // The main difference between 3.1.1 and >=3.1.2 is the improved eviction mechanism,
```

Comment:
Can you update the comment to match the discussion? You can mention KAFKA-18089. Thanks